There is no alternative language record (#110)
TYPO3-Documentation/TYPO3CMS-Guide-FrontendLocalization: Documentation/PageTranslations/Index.rst

Translating pages
=================

In order to translate a page in the page tree you create a

> copy of that record starting by its parent page. 

Reviewer: ```suggestion
copy of that page with a new language relation by using the module :guilabel:`Page > Language Comparison` and the dropdown :guilabel:`Create a new translation of this page`. 
```

I think we should then prevent the use of the term "record", and only refer to "page"?

And I think we need to describe the GUI method more properly? Otherwise if it just says "create a copy" people think they really should copy a page, and not using the proper "Create a new translation of this page" way.

Author: I think it would be helpful to mention here that the "Language Comparison" is a select box on the top of the right page. For me it has been hard to figure this out.

Reviewer: > I think it would be helpful to mention here that the "Language Comparison" is a select box on the top of the right page. For me it has been hard to figure this out. And it should be mentioned how the Select Box works. You need to choose the language and then it starts immediately.

I agree, do you think my proposed suggestion doesn't address this? Feel free to make an alternative wording suggestion.

> It will be generated in the default language with fields prefixed by a text similar to "[Translate to Language X:]". 

Reviewer: ```suggestion
It will be generated in the default language with all page property fields (like the title, description and others), prefixed by a text similar to "[Translate to Language X:]". 
```

Author: Don't mention "and others" here.

Reviewer: adjusted, feel free to apply

> The translated content needs to be put here instead of this prefix and the original text.

Reviewer: ```suggestion
The translated content needs to be put here, replacing the placeholder text with prefix and the default language contents.
```

Translating pages
=================

> In order to translate a page in the page tree just create a

Reviewer: ```suggestion
In order to translate a page in the page tree, create a
```

copy of it with a new language relation. Use the module :guilabel:`Page` and on the top of the right side 
:guilabel:`Language Comparison` and the lower dropdown :guilabel:`Create a new translation of this page`. 
As soon as you choose a language of this dropdown the translation process will start with the copy of this page.

> The page copy will be generated in the default language with page property fields like title or description prefixed by a text similar to "[Translate to Language X:]". 

Reviewer: ```suggestion
The page copy will be generated in the default language with page property fields like `title` or `description` prefixed by a text similar to "[Translate to Language X:]". 
```

> The translated content needs to be put here, replacing the placeholder text with prefix and 

Reviewer: ```suggestion
The translated content needs to be put here, replacing the placeholder text (including the prefix and 
```

> the copied default language contents.

Reviewer: ```suggestion
the copied default language contents).
```

    Editing the page fields for an alternative page language

> All of the fields are the same as for the page of the default language. Below

Reviewer: ```suggestion
Most of the fields are the same as for the page of the default language. Below
```

Actually the fields "backend_layout" and a few other ones are only available for the default language. So I think we need to keep "most" here.

Author: Well, mention that some fields are only visible for the default language.

Reviewer: I think that what "most" achieves here, I don't think we need to make it longer than before. Just important that "all" is not right.

the fields the content of the default language is shown.

> If a page has no translation,

Reviewer: ```suggestion
If a page has no translation meta-data,
```

Maybe this better reflects that it's about a "pages" database entry here, without resorting to use "record"?

Author: The term 'meta-data' is not understandable.

Reviewer: What do you suggest? Just "translation" is wrong, because here it's about a proxy entry for the language in "pages", and not the CONTENT of it. That's why I picked "meta-data", but maybe you have another idea?

Most of the fields are the same as for the page of the default language. Below
the fields the content of the default language is shown.

> As long as a page has no translated copy page,

Reviewer: ```suggestion
As long as a page has no translated page record,
```
